dashboard/src/shared/specs/mountWrapper.tsx

```diff
@@ -32,6 +32,7 @@ export const initialState = {
       "default-cluster": {
         currentNamespace: "default",
         namespaces: ["default", "other"],
+        canCreateNS: true,
```

Comment:
Just out of curiosity, why this default value is set as `true` whereas the one in cluster.ts is `false` ?

Reply:
the `cluster.ts` contains initial empty values. The empty value for booleans is `false`. But in this case, this `mountWrapper` is used in tests and it's mocking an already populated state (so it's not empty), and I set the value to `true` to maintain previous behavior (since before, the "Create Namespace" button was always enabled).